Command/RemoveThumbsCommand.php

```diff
@@ -146,10 +149,18 @@ public function getProvider()
     {
         $providerName = $this->input->getArgument('providerName');
         if (null === $providerName) {
+            $questionHelper = $this->getQuestionHelper();
```

Comment:
I'm quite :-1: about storing the question helper on a variable multiple time during the command.

Calling it directly from `$this` would be enough and clearer IMHO.